README.md

```
@@ -1,28 +1,125 @@
+        <img src="https://img.shields.io/badge/Mentor-F09BCD?style=for-the-badge"> <br> 
+        <a href="https://github.com/Tasin-BS">
+            <img src="https://img.shields.io/badge/GitHub-100000?style=flat&logo=github&logoColor=white"/>
+        </a>
+    </tr>
+  </table>
+</div>
+
+## 📸 Project Description
+Oakcan is a robust microservice-based file management platform designed for scalable and secure file operations. It enables efficient file upload/download, synchronization across devices, user management, and email-based notifications, making it ideal for cloud-native ecosystems.
+
+### Key Features
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)